stable/mongodb-replicaset/templates/_helpers.tpl

```diff
@@ -61,3 +61,18 @@ Create the name for the key secret.
         {{- template "mongodb-replicaset.fullname" . -}}-keyfile
     {{- end -}}
 {{- end -}}
+
+{{- define "mongodb-replicaset.connection-string"  -}}
+  {{- $string := "" -}}
+  {{- if .Values.auth.enabled }}
+   {{- $string = printf "mongodb://$METRICS_USER:$METRICS_PASSWORD@localhost:%s" (.Values.port|cat) -}}
```

Comment:
I'm not familiar with this cat command. Do you have any documentation on it? What does it do here that can't be accomplished with `printf "....%d" .Values.port -}}`?

Reply:
I had troube with using printf %d. It always lead to very strange output. cat in this case makes sure that the output is always interpreted as a string. Which worked for me in this situation.

`--mongodb.uri mongodb://$METRICS_USER:$METRICS_PASSWORD@localhost:%!!(MISSING)d(float64=27017)`

Reply:
Switched it to `toString` which is  a lot more self explanatory at least.